compiler-rt/lib/scudo/standalone/allocator_config.h

```diff
@@ -146,9 +146,11 @@ struct FuchsiaConfig {
     // Support 39-bit VMA for riscv-64
     static const uptr RegionSizeLog = 28U;
     static const uptr GroupSizeLog = 19U;
+    static const bool EnableContiguousRegions = false;
 #else
     static const uptr RegionSizeLog = 30U;
     static const uptr GroupSizeLog = 21U;
+    static const bool EnableContiguousRegions = true;
```

Comment:
The default is true, so it doesn't seem you would need to explicitly set this to true.

Reply:
Removed, thanks!